Datadog/E2ETests/RUM/RUMMonitorE2ETests.swift

```diff
@@ -0,0 +1,962 @@
+/*
+ * Unless explicitly stated otherwise all files in this repository are licensed under the Apache License Version 2.0.
+ * This product includes software developed at Datadog (https://www.datadoghq.com/).
+ * Copyright 2019-2020 Datadog, Inc.
+ */
+
+import Foundation
+import Datadog
+
+class RUMMonitorE2ETests: E2ETests {
+    private lazy var rum = Global.rum.dd
+
+    let actionTypePool = [RUMUserActionType.swipe, .scroll, .tap, .custom]
+    let nonCustomActionTypePool = [RUMUserActionType.swipe, .scroll, .tap]
+
+    /// - data monitor:
```

Comment:
We're missing `-api-surface:` for this and some other tests. That will be required later to compute API coverage against `api-surface-swift` ([like we do in Android SDK](https://github.com/DataDog/dd-sdk-android/blob/master/instrumentation-tools/nightly_tests_code_coverage.py)).